pkg/local_object_storage/blobstor/fstree/fstree_write_generic.go

```diff
@@ -46,7 +46,7 @@ func (t *FSTree) writeData(p string, data []byte) error {
 	}
 
 	// unreachable, but precaution never hurts, especially 1 day before release.
-	return fmt.Errorf("couldn't read file after %d retries", retryCount)
+	return fmt.Errorf("couldn't write file after %d retries", retryCount)
```

Comment:
wow, more like a bugfix for a separate commit

Reply:
this is not a bug, just typo

Reply:
logging "read" while writing is a bug to me. do not insist on a separate commit much